System.Device.I2c/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following 
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("CSharp.BlankApplication")]
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("")]
+[assembly: AssemblyProduct("CSharp.BlankApplication")]
+[assembly: AssemblyCopyright("Copyright (c) .NET Foundation and Contributors")]
+[assembly: AssemblyTrademark("")]
+[assembly: AssemblyCulture("")]
+
+////////////////////////////////////////////////////////////////
+// update this whenever the native assembly signature changes //
+[assembly: AssemblyNativeVersion("100.2.0.2")]
```

Comment:
should this be something like 1.0.0.0? or match what we have in IoT?

Reply:
This attribute is required to check if a firmware image has support for what's about to be deployed.

See [this](https://www.nanoframework.net/nuget-assembly-and-native-versions/) blog post for a detailed explanation. 
Happy to explain it in more detail if you want. 😄

Reply:
@Ellerbach this should start at 100.1.0.0, please